### ISSUE TITLE
tsclean from forecast package and lambda argument

### ISSUE THREAD

**Issue by DevKiddo** [robjhyndman/forecast#190]
Dear Rob,

Not exactly a technical issue, but perhaps you can help me understand this part of your package:

http://stats.stackexchange.com/questions/183806/tsclean-from-forecast-package-and-lambda-argument

Thank you,
Jimmy

**Comment by DevKiddo** [comment]
this article helped a lot! http://www.sigmamagic.com/forum/archives/297

**Comment by DevKiddo** [comment]
Now that I understand better what the Box Cox transformation parameter is and how to estimate the best Box Cox transformation parameter for tsclean, does forecast come with any MLE function to get the best value for lambda programatically?

**Comment by robjhyndman** [comment]
Try the `BoxCox.lambda()` function.

**Comment by DevKiddo** [comment]
Hi Rob, thank you for your reply. I tried it and it returned a 0.4 parameter which when applied to tsclean(), didn't do anything to my dataset. The outliers remained there and there were no transformation. On the other hand, when trying the parameter 4.6 fore example (one that I found after testing with values ranging from -5,5), tsclean() got rid of all my outliers and transformed them. Any idea of what might be happening?
